node/Cargo.toml

```diff
@@ -17,57 +17,68 @@ targets = ['x86_64-unknown-linux-gnu']
 [dependencies]
 clap = { version = "3.1.18", features = ["derive"] }
 bs58 = "0.4.0"
+protoc-rust = "2.28.0"
```

Comment:
Is this necessary? I know it was added to a bunch of other places in error

Reply:
I think this can go now.